lib/launch/launch_configs.dart

```diff
@@ -38,6 +42,12 @@ class LaunchConfigurationManager implements Disposable, StateStorable {
     return _getCreateProjectConfig(path).getConfigs();
   }
 
+  List<LaunchConfiguration> getAllConfigs() {
```

Comment:
I think `get` for things that are inexpensive to call, like O(1) amortized. This could be a `get`ter as well -